### DISCUSSION TITLE
Library playlist functionality

### DISCUSSION THREAD

**Discussion by 216598762** [TT-ReBORN/Georgia-ReBORN#103] [Q&A] [answered]
Adore the theme, but it seems to take away one of the critical features of foobar I love most. 

I usually keep an auto playlist of my music and shuffle that; if I want to listen to a particular song, I'd click on it from the left bar or the file selection; once I finish that song, it goes back to reshuffling the auto playlist. Is this possible in reborn, or is the way the library view is scripted making it impossible?

I've gone through every setting and am unsure if I'm missing something.

**Comment by TT-ReBORN** [comment]
Hi,

I don't know if I understand you correctly, but you could achieve that by first setting the playback order button in the lower bar to shuffle mode and then when you are on the playlist select the songs you want to listen to via right click for context menu > Add to playback queue. You will notice that the selected song now has a queue with a number, you can do this for other songs too, meaning - 
when the current song ends, it will automatically play the song from the queues. 
When all queues have been played, it will resume random shuffle.

-TT

**Reply by 216598762** [reply]
I suppose it's more of an issue of wanting to change my double click action to do that, rather than have to open a context menu. I see reborn has the capability in the options but it only has the three otpions.
![image](https://user-images.githubusercontent.com/69487959/221601528-d0b60504-b973-455b-89c9-6ef985bc4bb8.png)

**Comment by 216598762** [comment]
I've used shuffle, not sure if we're talking past each other. When I double click it replaces the whole library playlist with one song, and puts that on repeat.

**Comment by TT-ReBORN** [comment, marked as answer]
Yes that was a misunderstanding. So you do know how the Library works? Did you first read the FAQ here?
https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/58#discussion-3894207

That means you also know how the Playlist system works, what the exclusive autoplaylist "Library View" does.
If you know all that, and your "Library View" has populated tracks and you want randomly play a song from the Library ( not the Playlist )
just right click on the song and select "Play", it will temporary play the song and after it is finished, it will resume shuffle play from your "Library View" autoplaylist.

-TT

**Reply by 216598762** [reply]
Yeah I've gotten to that point of workaround, was just curious if it was possible to make that the double click action like foobar normally acts. No worries if not, just gotta get used to it.

Potential concepts to fix this, if interested would be adding that to the context menu as a double click interaction, or allowing you to lock the playlist once you've selected tacks, It's really just preference due to habit.

**Reply by TT-ReBORN** [reply]
So you want to have an extra double click action option for the Library that behaves the same as, right clicking on track from the Library and selecting "Play"? That way it just plays the song and will not modify ( clear content and only adds selected ) the Library autoplaylist "Library View"? If so, it's no problem and will be available in Beta 11c.

**Reply by 216598762** [reply]
Yes, that would be perfect 😄 Thank you.
